experiment/molecule/src/common.ts

```diff
@@ -1,4 +1,5 @@
 import { BI } from "@ckb-lumos/bi";
+import * as toolkit from "@ckb-lumos/toolkit";
```

Comment:
it would be inappropriate to import `toolkit` just to convert `HexString` to `ArrayBuffer`, which can be implemented with a simple function